src/dataExplorer/components/FluxBrowser.tsx

```diff
@@ -0,0 +1,82 @@
+import React, {FC, useCallback, useContext, useEffect, useMemo} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+
+// Components
+import SelectorTitle from 'src/dataExplorer/components/SelectorTitle'
+import FilterList from 'src/shared/components/FilterList/FilterList'
+import FluxDocsTooltipContent from 'src/shared/components/functions/perFunction/FluxDocsTooltipContent'
+import Fn from 'src/shared/components/FilterList/MinimalistInjectOption'
+
+// State
+import {getFluxPackages} from 'src/shared/actions/fluxDocs'
+import {getAllFluxFunctions} from 'src/shared/selectors/app'
+import {EditorContext} from 'src/shared/contexts/editor'
+
+// Types
+import {FluxFunction} from 'src/types/shared'
+
+// Utils
+import {event} from 'src/cloud/utils/reporting'
+import {getFluxExample} from 'src/shared/utils/fluxExample'
+import {sortFuncs} from 'src/shared/components/functions/utils'
+import './FluxBrowser.scss'
+
+const TOOLTIP = `The flux standard library contains several packages, \
+functions, and variables which may be useful when constructing your flux query.`
+
+const FluxBrowser: FC = () => {
```

Comment:
why do we have three versions of this interaction? are they that different?

Reply:
Yes, they are different.
* Each of the three component uses the same child components (e.g. FilterList).
* But then has different grandchildren components (e.g. renderItem) which is provided in this higher level component.
  * The other 2 components have moved closer together over time, but this new FluxBrowser moves further away per the new design.
* This parent components is just the callback and the renderItem.

Reply:
The handoff was a Figma design.
I can double check (ask to message up the tree).

Reply:
oh yeah. a lot can be lost in just a picture. thanks for asking